src/components/notecontainer/notecontainer.jsx

```diff
@@ -1,7 +1,61 @@
 import "./notecontainer.css";
 import parse from "html-react-parser";
+import { useAuth, useNote } from "context";
+import axios from "axios";
+import { useLocation } from "react-router-dom";
+
+export const NoteContainer = ({ title, text, color, date, time, id }) => {
+  const { pathname } = useLocation();
+  const note = { title, text, color, date, time, id };
+  const { noteArrayDispatch } = useNote();
+  const { token } = useAuth();
+
+  const addArchiveNote = async (note) => {
+    try {
+      const response = await axios.post(
+        `/api/notes/archives/${id}`,
+        { note },
+        { headers: { authorization: token } }
+      );
+      noteArrayDispatch({
+        type: "ADD_TO_ARCHIVE",
+        payload: response.data,
+      });
+    } catch (e) {
+      console.log(e);
```

Comment:
remove all the logs before pushing, you can use toastify and give user an error message if something goes wrong

Reply:
I will add toastify component at last